samples/Kmd.Logic.DocumentGeneration.Client.GenerationSample/appsettings.json

```diff
@@ -2,11 +2,11 @@
   "TokenProvider": {
     "ClientId": "00000000-0000-0000-0000-000000000000",
     "ClientSecret": "",
-    "AuthorizationScope": "https://logicidentityprod.onmicrosoft.com/00000000-0000-0000-0000-000000000000/.default",
+    "AuthorizationScope": "https://logicidentityprod.onmicrosoft.com/00000000-0000-0000-0000-000000000000/.default"
   },
   "DocumentGeneration": {
     "SubscriptionId": "00000000-0000-0000-0000-000000000000",
-    "DocumentGenerationServiceUri": "https://gateway.kmdlogic.io/document-generation/v2",
```

Comment:
This is not the same v2.
This one is the version of the API in APIM.

Reply:
Fixed in 01522a4eaf410099e003bcc977431c76fee743e4